build.gradle.kts

```diff
@@ -30,7 +30,7 @@ allprojects {
 }
 
 subprojects {
-  if (name != "keel-bom") {
+  if (name != "keel-bom" && name != "keel-api") {
```

Comment:
Why do you want to exclude this from the BOM?

Reply:
That's not what this is doing. This is opting the `keel-api` module out of including kotlin and all of these dependencies from it.